src/gtk/toga_gtk/widgets/detailedlist.py

```diff
@@ -0,0 +1,55 @@
+from gi.repository import Gtk
+
+from .base import Widget
+
+class DetailedList(Widget):
+	def create(self):
+		self.native = Gtk.ScrolledWindow()
+		self.native.interface = self.interface
+
+		#Craete DetailedList
+		self.vbox = Gtk.VBox()
+
+		self.native.add_with_viewport(self.vbox)
```

Comment:
Looks like you're using tabs for indentation here. Pybee style is 4 spaces (I'm surprised this ins't picked up by linting..)

Reply:
I'm used to using tab spaces while doing indentation 😓  - But I had noticed that the project used 4 spaces and I'd set up a small linting tool with Sublime to do that for me. (Two linters to be looked into :P) - I'll make that change manually for now.